google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -205,4 +212,21 @@ public void testDeleteRules() {
       assertEquals(delRule, DeleteRule.fromPb(delRule.toPb()));
     }
   }
+
+  @Test
+  public void testLifecycleRules() {
+    Rule deleteLifecycleRule = new LifecycleRule(new LifecycleRule.DeleteLifecycleAction(),
+            new LifecycleRule.LifecycleCondition.Builder().setAge(10).build()).toPb();
```

Comment:
Change the way a builder is created.
`newBuilder()` instead to follow along with:
https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java#L1044

Similarly if a developer wants to modify the state of the Rule, add `toBuilder()`:
https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java#L912

This will help remove the need to use `new`. Looks cleaner and consistent with the rest of the surface.